client/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -248,6 +248,18 @@ describe('FeedbackGeneratorService', function() {
     });
   });
 
+  describe('_getUnfamiliarLanguageFeedback', function() {
+    it('should return the string for the text feedback', function() {
+      var feedbackString = FeedbackGeneratorService
+        ._getUnfamiliarLanguageFeedback();
+      expect(feedbackString).toEqual([
+        "Seems like you're having some trouble with this language. Why ",
+        "don't you take a look at the New to Python button at the bottom ",
```

Comment:
Small nit with wording but rephrase to clarify looking at the "primer linked through the `New to Python` button" rather than just the button itself